Frogies-Project/Assets/Scripts/Items/Enum/ItemId.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Items.Enum
+{
+    public enum ItemId
+    {
+        None = 0,
+        // Weapons
+        WoodenSword = 1,
+        WoodenShield = 2,
+        // Armor
+        LeatherHelmet = 3,
+        LeatherChestplate = 4,
+        LeatherLeggings = 5,
+        LeatherBoots = 6,
+        // Consumables
+        HealthPotion = 7,
+        ManaPotion = 8,
```

Comment:
do we need all of this? I thought we decided on: bottle (empty, full), coins, flies (health potion)

Reply:
As a weapon we can add something like rock/seed/nut to throw in an enemy. Also something like mushrooms can be added as mana potion (so we can keep it also)